src/http/httpHeaders.ts

```diff
@@ -170,4 +170,4 @@ export const AUTHORIZATION_HEADER = 'authorization';
 export const FORM_URLENCODED_CONTENT_TYPE = 'application/x-www-form-urlencoded';
 export const JSON_CONTENT_TYPE = 'application/json';
 export const TEXT_CONTENT_TYPE = 'text/plain; charset=utf-8';
-export const XML_CONTENT_TYPE = 'application/xml';
+export const XML_CONTENT_TYPE = 'application/xml; charset=utf-8';
```

Comment:
```suggestion
export const XML_CONTENT_TYPE = 'application/xml';
```
I dont think this change is needed since we are declaring the content type in the `modifyCallBxml`function and not inferring from `req.callAs*` due to the fact that this is sending XML and returning JSON